backup/setup/migrations.json

```diff
@@ -601,5 +601,23 @@
 		"created_at": {
 			"$date": "2025-07-13T22:23:24.480Z"
 		}
+	},
+	{
+		"_id": {
+			"$oid": "6880f5b9457800dab3bf11f0"
+		},
+		"name": "Migration20250708112351",
```

Comment:
was this added here as well? somehow I don't get githubs style of showing changes here

Reply:
Fixed. Could not find the corresponding Migration - so I removed it.